sound/soc/sof/intel/mtl.c

```diff
@@ -755,8 +755,9 @@ int sof_mtl_ops_init(struct snd_sof_dev *sdev)
 	/* parse platform specific extended manifest */
 	sof_mtl_ops.parse_platform_ext_manifest = NULL;
 
-	/* dsp core get/put */
 	/* TODO: add core_get and core_put */
+	sof_mtl_ops.core_get = mtl_dsp_core_power_up,
+	sof_mtl_ops.core_put = mtl_dsp_core_power_down,
```

Comment:
I think this is incorrect. core_get should involve 2 things: 1. powering up the core if host managed 2. sending the IPC to the DSP. same with core_put.